index.html

```diff
@@ -0,0 +1,179 @@
+    
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/para-gliding.jpeg" alt="Para Gliding">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Para Gliding</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Para Gliding will be performed during opening ceremony by our instructors
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/rock-logo.jpeg" class="card-img" alt="Rock Climb">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Rock Climbing</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Rock Climbing competition will be held and judged by
+            our trainers and Instructors
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/trophy.jpeg" class="card-img" alt="Rock Climber">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Winner Prize</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Winners of the Rock Climbing competition will be awarded 
+            cash prize, certificate and a trophy by H.R.H Abu Talha 
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/Dance.jpeg" class="card-img" alt="Dance and Music">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Closing Ceremony</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Closing Ceremony will include concert with our Traditional dance and Local Music
+          </p>
+        </div>
+      </div>
+    </div>
+    <button type="button" class="main-program-btn hideOnDesktop-I fc-white"> Join GB Rock Climbing Fixture 2023</button>
+    <a href="#" class="main-program-link hideOnMobile-f fc-white">SEE THE WHOLE PROGRAM</a>
+  </section>
+
+  <section class="Featured-Speakers" id="Guest-Container">
+    <h3 class="h3 trainers">Our Trainers
+      <span class="title-underline"></span>
+    </h3>
+    <div class="card">
+        <div class="card-header">
+          <img src="./images/speaker_01.png" class="card-img" alt="Mr. Anderson">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Mr. Anderson</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Rock Climbing Instructors and C.E.O of the institution. He is a professional
+            Mountaineer and a good singer.
+          </p>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/speaker_02.png" class="card-img" alt="Mrs. Anderson">
+        </div>
+        <div class="card-title">
```

Comment:
- [ ] Please make sure you generate this page dynamically using `js` as per the requirement, remove all these hard-coded lines from the HTML, and create an array of objects with speakers data that will be rendered on page load using javascript

<img width="1022" alt="1" src="https://github.com/AbuTalha3/Capstone-project-Module-One/assets/77843050/3451125b-d74e-484d-accf-15ee3c0e6dd5">